packages/beacon-state-transition/src/cache/epochContext.ts

```diff
@@ -389,7 +418,10 @@ export class EpochContext {
     const nextEpoch = currEpoch + 1;
 
     this.nextShuffling = computeEpochShuffling(state, epochProcess.nextEpochShufflingActiveValidatorIndices, nextEpoch);
-    this.proposers = computeProposers(state, this.currentShuffling, this.effectiveBalanceIncrements);
+    this.currentProposerSeed = getSeed(state, this.currentShuffling.epoch, DOMAIN_BEACON_PROPOSER);
+    this.nextProposerSeed = getSeed(state, this.nextShuffling.epoch, DOMAIN_BEACON_PROPOSER);
```

Comment:
we only need to call `getSeed()` once for `this.nextProposerSeed`, `this.currrentProposerSeed = this.nextPrroposerSeed`

Reply:
The JSDoc for `nextProposerSeed` must be placed in the variable declaration ([here](https://github.com/ChainSafe/lodestar/pull/3782/files#diff-3e6d2b0cf933b99848484935101c7016da5a077e85ca7e16fa132d86a202ab91R156)), here it has no effect. You can duplicate the comment here too if appropiate but use regular `//` comment syntax. Also try to fit the comment to 120 wide characters for consistent formatting.

Reply:
Also here you can comment (with `//`) that the cost of getSeed is negligible provided that this path runs once per epoch.